index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <nav class="main-nav">
+            <div class="left-bar">
+                <p class="time">9:41</p>
+                <p class="name">Oscar Byiringiro</p>
+            </div>
+            <div class="right-bar">
+                <div class="top-icons">
+                    <img src="./images/Cellular Connection.png">
+                    <img src="./images/Wifi.png">
+                    <img src="./images/Battery.png">
+                                
+                            
+                </div>
+                <div class="bottom-icons">
+                    <img src="./images/Union.png">
+                        
+                </div>
+                
+                    
+            </div>
+        </nav>
+        <section class="main-body">
+            <div class="container">
+                <div class="intro">
+                    <p>Hey There</p>
+                    <p>I'm Oscar</p>
+                </div>
+                <div class="title">
+                    <p>I'am a software developer</p>
+                </div>
```

Comment:
- [ ] As suggested by previous reviewer, please make sure to use semantic `<hx>` tags for your main headings. `<hx>` tags can help boost a web page's search engine rankings and make it more discoverable. And we want that to happen. right ? 🎯